src/generator-utilities.js

```diff
@@ -152,7 +153,16 @@ function createTemplate(directory, placeholderNames, omitComments, callback) {
             data = removeComments(data);
         }
 
-        fs.writeFile(directory.generated, data, err => {
+        const prettierParser = directory.generated.indexOf('.scss') > -1 ? 'scss' : 'babylon';
+
+        const formattedCode = prettier.format(data, {
+            tabWidth: 4,
+            singleQuote: true,
+            printWidth: 100,
+            parser: prettierParser
+        });
+
+        fs.writeFile(directory.generated, formattedCode, err => {
```

Comment:
Nice work dude!

Reply:
@JPStrydom thanks, dude. I like the config feature you added. I'm thinking of moving default settings to config and allow advanced users to specify their prettier settings. Do you think it gonna be overkill?

Reply:
I think we can take it even one step further... We can allow users to specify their own templates as well. We already have the `grcc.json` file that we can use to specify all kinds of config. But I think it's a nice to have. Once users start requesting it we can add it. What do you think?